docs/secure-services.md

```diff
@@ -107,7 +107,8 @@ A new Docker controller image has to be built to add the formerly modified
 included in this template (njs module & `oauth2_token_introspection` folder).
 
 You just need to follow the instructions in the `ingress-nginx` repository and
-run something similar to the following:
+review the [development guidelines](https://kubernetes.github.io/ingress-nginx/development/).
```

Comment:
broken link

Reply:
https://kubernetes.github.io/ingress-nginx/development/
Just access this url and it opens with following page:
https://ibb.co/8z24JLY

Reply:
true, works